esp/src/eclwatch/DynamicESDLDefinitionQueryWidget.js

```diff
@@ -0,0 +1,111 @@
+/*##############################################################################
+#    HPCC SYSTEMS software Copyright (C) 2012 HPCC Systems®.
```

Comment:
Date wrong and I don't think we need to copyright header once the licence file is present in the repo.